### PR TITLE
Upload files

### PR DESCRIPTION
**Pull request by yankun**
Added the possibility to upload files to Gitlab, so that Issues or other places using markdown can be enriched with attachments, by using the Markdown property provided by the upload response.

### CODE REVIEW

**Comment by nmklotas** [comment]
Thank you! Looks nice.

**Comment by yankun** [comment]
Your welcome! Hope to see 1.0.5 on NuGet soon :)

**Comment by yankun** [comment]
By the way, this could close issue https://github.com/nmklotas/GitLabApiClient/issues/20.

**Comment by nmklotas** [comment]
Thanks, yes I've created 1.0.5 ~22 hours ago

File: src/GitLabApiClient/Models/Uploads/Requests/CreateUploadRequest.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+namespace GitLabApiClient.Models.Uploads.Requests
+{
+    /// <summary>
+    /// A upload (file) for Gitlab, which can be embedded via markdown
+    /// </summary>
+    public sealed class CreateUploadRequest
+    {
+        public CreateUploadRequest(string projectId, Stream stream, string fileName)
+        {
+            projectId = projectId;
```
Comment (splitt3r) [inline src/GitLabApiClient/Models/Uploads/Requests/CreateUploadRequest.cs]: This should be `ProjectId` shouldn´t it?

File: src/GitLabApiClient/UploadsClient.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        private readonly GitLabHttpFacade _httpFacade;
+
+        internal UploadsClient(GitLabHttpFacade httpFacade) =>
+            _httpFacade = httpFacade;
+
+        /// <summary>
+        /// Uploadses a file for the provided project. 
+        /// </summary>
+        /// <param name="projectId">The id of the project</param>
+        /// <param name="uploadRequest">The upload request containing the filename and stream to be uploaded</param>
```
Comment (splitt3r) [inline src/GitLabApiClient/UploadsClient.cs]: The documentation doesn´t match the actual code.

**Comment by splitt3r** [comment]
Awesome work 👍

**Pull request merged**